common/src/test/java/io/netty/util/NetUtilTest.java

```diff
@@ -87,7 +92,11 @@
             "[2001:0000:4136:e378:8000:63bf:3fff:fdd2]", "200100004136e378800063bf3ffffdd2",
             "0:1:2:3:4:5:6:789a",                        "0000000100020003000400050006789a",
             "0:1:2:3::f",                                "0000000100020003000000000000000f",
-            "0:0:0:0:0:0:10.0.0.1",                      "0000000000000000000000000a000001",
+            "0:0:0:0:0:0:10.0.0.1",                      "00000000000000000000ffff0a000001",
+            "0:0:0:0:0::10.0.0.1",                       "00000000000000000000ffff0a000001",
+            "::0:0:0:0:0:10.0.0.1",                      "00000000000000000000ffff0a000001",
+            "0:0:0:0:0:0::10.0.0.1",                     "00000000000000000000ffff0a000001",
```

Comment:
I'm about of count of zeros and "::" after. There can not be more bits in place of "::".
This is confirmed by examples [1] and some online-validators: [2] and [3].

[1] https://tools.ietf.org/html/rfc4291#section-2.2
[2] http://formvalidation.io/validators/ip/ ("Try it" in the end)
[3] https://www.helpsystems.com/intermapper/ipv6-test-address-validation